### DISCUSSION TITLE
Scripts in Z2m

### DISCUSSION THREAD

**Discussion by AndRe5575** [Koenkk/zigbee2mqtt#22159] [General]
Hi together,

hope someone is able to help me. I want to move from DeConz to Z2m. At the moment I´m using the WakeUp light screnes from DeConz which I also want to use in the future. Is it possible to create this scenes also with Z2M or do I have to build this function in OpenHab or ioBroker or simillar?

AndRe

**Comment by francisp2** [comment]
https://github.com/Luligu/zigbee2mqtt-automations

**Reply by AndRe5575** [reply]
Looks good, thank you very much. Now I just have to check how to change the triggers via a nice frontend (time/date/on/off/etc.)
